proto/messages.proto

```diff
@@ -48,8 +48,9 @@ message Ephemeral {
 
 message Text {
   required string content = 1;
-  repeated Mention mention = 2;
+  //repeated Mention mention = 2; // Deprecated, use Mention = 4
   repeated LinkPreview link_preview = 3;
+  repeated Mention mention = 4;
```

Comment:
As this is a repeated field this should be `repeated Mention mentions = 4` naming wise

Reply:
Also we will need to rename the `Mention` type now as we changed it, otherwise this will go against the rules when removing fields:

> Don't change the field numbers for any existing fields.